apps/application-system/form/src/environments/environment.ts

```diff
@@ -10,6 +10,7 @@ const devConfig = {
     dsn:
       'https://22093678b2b24a0cad25111c1806a8d7@o406638.ingest.sentry.io/5530607',
   },
+  featureFlagSdkKey: 'YcfYCOwBTUeI04mWOWpPdA/KgCHhUk0_k2BdiKMaNh3qA',
```

Comment:
I assume this is intentional?

Reply:
I copied this from the service-portal. Not sure if this is sensitive or if we need to remove this from the codebase and cycle the key @sindrig?

Reply:
I don't remember if it was ever discussed. IIRC the only things you can do with this key is get dev feature flag statuses, and since our repo is open source I doubt there's an attack vector there.

Reply:
@sindrig while you're here, can you do that devops review? ;)